tests/kerastuner/tuners/sklearn_test.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2019 The Keras Tuner Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+from sklearn import datasets
+from sklearn import ensemble
+from sklearn import linear_model
+from sklearn import metrics
+from sklearn import model_selection
+
+import kerastuner as kt
+from kerastuner.tuners import sklearn as sklearn_tuner
+
+
+def build_model(hp):
+    model_type = hp.Choice('model_type', ['random_forest', 'ridge'])
+    if model_type == 'random_forest':
+        with hp.conditional_scope('model_type', 'random_forest'):
+            model = ensemble.RandomForestClassifier(
+                n_estimators=hp.Int('n_estimators', 10, 50, step=10),
+                max_depth=hp.Int('max_depth', 3, 10))
+    elif model_type == 'ridge':
+        with hp.conditional_scope('model_type', 'ridge'):
+            model = linear_model.RidgeClassifier(
+                alpha=hp.Float('alpha', 1e-3, 1, sampling='log'))
+    else:
+        raise ValueError('Unrecognized model_type')
+    return model
+
+
+@pytest.fixture(scope='function')
+def tmp_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp('hyperband_test', numbered=True)
+
+
+def test_sklearn_tuner_simple(tmp_dir):
+    tuner = sklearn_tuner.Sklearn(
+        oracle=kt.oracles.BayesianOptimization(
+            objective=kt.Objective('score', 'max'),
+            max_trials=10),
+        hypermodel=build_model,
+        directory=tmp_dir)
+
+    X = np.random.uniform(size=(50, 10))
```

Comment:
Unlike the sklearn convention, as per PEP8 we should use lowercase variables

Reply:
Done